### PR TITLE
✂️✂️✂️ Drop PWA

### PR DESCRIPTION
**Pull request by ludwiczakpawel**
- [x] Refactor

## Description

This PR removes PWA buttons from header and PWA-related code. More details on reasoning [here](https://forem-team.slack.com/archives/C7GE84DEJ/p1638453406266600).

## Related Tickets & Documents

- [Slack thread](https://forem-team.slack.com/archives/C7GE84DEJ/p1638453406266600).

## QA Instructions, Screenshots, Recordings

Not sure if there's a way to test it per se. Definitely take a look at code removals. 

### UI accessibility concerns?

No.

## Added/updated tests?

No.

## [Forem core team only] How will this change be communicated?

It has already been announced in few places I believe.

### CODE REVIEW

File: app/assets/javascripts/initializers/runtime.js [hunk 1]
```diff
@@ -20,16 +20,13 @@ class Runtime {
   /**
    * This function returns a string that represents the current Medium where
    * the app is currently running. The currently supported mediums are Browser,
-   * ForemWebView and PWA.
+   * and ForemWebView.
    *
    * @returns {String} One of the supported Mediums or 'Unsupported'
    */
   static currentMedium() {
```
Comment (aitchiss) [inline app/assets/javascripts/initializers/runtime.js:27]: (suggestion / not blocking) now there are only two options we could simplify this if we want. Something like

```js
return /ForemWebView/i.test(navigator.userAgent) ? 'ForemWebView' : 'Browser';
```

File: app/assets/javascripts/initializers/runtime.js [hunk 2]
```diff
@@ -20,16 +20,13 @@ class Runtime {
   /**
    * This function returns a string that represents the current Medium where
    * the app is currently running. The currently supported mediums are Browser,
-   * ForemWebView and PWA.
+   * and ForemWebView.
    *
    * @returns {String} One of the supported Mediums or 'Unsupported'
```
Comment (aitchiss) [inline app/assets/javascripts/initializers/runtime.js:25]: It's not related to this PR, but it's not correct at the moment for the logic we have 😅 

```suggestion
   * @returns {String} One of the supported Mediums 
```

File: app/assets/stylesheets/config/_generator.scss [hunk 3]
```diff
@@ -453,7 +453,7 @@
     null
   ),
   ('object', 'object-fit', ('cover': cover), (), false),
-  ('aspect', 'aspect-ratio', ('16-9': 16 / 9), (), false),
+  ('aspect', 'aspect-ratio', ('16-9': calc(16 / 9)), (), false),
```
Comment (aitchiss) [inline app/assets/stylesheets/config/_generator.scss:456]: This is covered in your CSS generator PR, which I guess we're aiming to merge first

**Review by aitchiss — APPROVED** [review]
Looks good to me - nice to see this getting cleaned up 🧹 ✨

File: app/assets/stylesheets/config/_generator.scss (hunk 3 above)
Comment (ludwiczakpawel) [reply app/assets/stylesheets/config/_generator.scss]: indeed, ive just merged it, so resolving this one.

File: app/assets/javascripts/initializers/runtime.js (hunk 2 above)
Comment (ludwiczakpawel) [reply app/assets/javascripts/initializers/runtime.js:25]: I'll sneak it in.

**Review by nickytonline — APPROVED** [review]
✂️✂️✂️🚀

File: app/assets/javascripts/initializers/runtime.js (hunk 1 above)
Comment (ludwiczakpawel) [reply app/assets/javascripts/initializers/runtime.js:27]: done!

**Pull request merged**